### PR TITLE
updated tree

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

File: create_framework/separation-of-concerns.rst [hunk 1]
```diff
@@ -150,6 +150,7 @@ To sum up, here is the new file layout:
 
     example.com
     ├── composer.json
+    ├── composer.lock    
     │   src
```
Comment (javiereguiluz) [inline create_framework/separation-of-concerns.rst]: This line should be:

```
├── src
```

Instead of:

```
│   src
```

Comment (OskarStark) [inline create_framework/separation-of-concerns.rst]: :+1:

**Comment by WouterJ** [comment]
Thanks for your review & PRs for this new tutorial series Oskar! I've patched this one in 2.3 as well.

**Comment by OskarStark** [comment]
Thank you @WouterJ

**Pull request closed without merging**